src/components/PeopleFilters.tsx

```diff
@@ -1,18 +1,56 @@
+import { Link, useSearchParams } from 'react-router-dom';
+import classnames from 'classnames';
+import { getSearchWith, SearchParams } from '../utils/searchHelper';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+  const sex = searchParams.get('sex') || '';
+
+  function setSearchWith(params: SearchParams) {
+    const search = getSearchWith(searchParams, params);
+
+    setSearchParams(search);
+  }
+
+  function getCenturies(min: number, max: number) {
+    const centuryNumbers = [];
+
+    for (let i = min; i <= max; i++) {
+      centuryNumbers.push(i);
+    }
+
+    return centuryNumbers;
+  }
+
+  function handleQueryChange(e: React.ChangeEvent<HTMLInputElement>) {
+    setSearchWith({ query: e.target.value || null });
+  }
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
```

Comment:
the best use of BEM here would be panel__tabs (i think)
the exercise might be a little outdated